tests/test_e2e.py

```diff
@@ -135,3 +138,35 @@ def test_permissions(self, user_uuid):
         for other_user_uuid, source_document_file_uuids in TestEndToEnd.source_document_file_uuids.items():
             if other_user_uuid != user_uuid:
                 assert TestEndToEnd.file_uuids[user_uuid] not in source_document_file_uuids
+
+    @pytest.mark.asyncio()
+    async def test_streaming_rag(self):
```

Comment:
when we have fixed authorization in websockets (another PR) we will be able to paparmetrize this for different users